docs/_docs/components/bar.md

```diff
@@ -165,7 +158,135 @@ Right-aligned vertical bar example:
       </Bar>
     </LayoutSiderContent>
   </LayoutSider>
+  <Layout>
+    <LayoutHeader Fixed="true">
+      Header
+    </LayoutHeader>
+    <LayoutContent>
+      Content
+    </LayoutContent>
+  </Layout>
+</Layout>
+```
+
+### Vertical Bar Toggler
+
+The `BarToggler` for Vertical Bar supports 2 modes:
+- `BarTogglerMode.Normal` (default)
+- `BarTogglerMode.Popout`
+
+These 2 modes, along with an external implementation, provides 3 easy to configure implementations.
+
+**1. Inline**
+
+By simply adding the `BarToggler` anywhere inside the Vertical `Bar` component (at the top level), you will have an inline toggler added to your bar at this location.
+
+Example (used by demo application):
+```html
+<Bar Mode="BarMode.VerticalInline">
+  <BarToggler />
+  <BarBrand>
+    ...
+  </BarBrand>
+  <BarMenu>
+    ...
+  </BarMenu>
+</Bar>
+```
+
+**2. Popout**
+
+Once again, by simply adding the `BarToggler` anywhere inside the Vertical `Bar` component (at the top level), and setting the `Mode` to `Popout` you will have an popout toggler added to your bar at this location.
+
+Example:
+```html
+<Bar Mode="BarMode.VerticalInline">
+  <BarBrand>
+    ...
+  </BarBrand>
+  <BarToggler Mode="BarTogglerMode.Popout" />
+  <BarMenu>
+    ...
+  </BarMenu>
+</Bar>
+```
+
+**3. Extneral** (controlled by Top Bar)
```

Comment:
Small typo :)

Reply:
Thanks for all the typo fixes! 
I was doing this on my laptop.. not used to the keyboard 😂

Reply:
Haha np, you did most of the things anyways, so I should be the one giving the thanks :)